azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -550,12 +550,12 @@ private boolean isContainerizedDispatchMethodEnabled() {
   private void prepareAndStartServer() throws Exception {
     this.executorManagerAdapter.start();
     this.executionLogsCleaner.start();
+    this.missedSchedulesManager.start();
```

Comment:
Start of missedScheduleManager is already consider the disabled case. it does not do anything if so.
It can be moved to TriggerManager. I hope to decouple those threads into a standalone position and a unified place to check the start/stop process in web server instead of coupling it into another busy thread. 
TriggerManager itself currently is not in a good design, so it does not gracefully shutdown. I have the alternative designs to migrate TriggerManager into Quartz and take that chance to refactor it.